src/runtime/particle-execution-context.ts

```diff
@@ -254,6 +258,10 @@ export class ParticleExecutionContext {
     }];
   }
 
+  private async rebootParticle(id: string) {
+    //TODO: implement
```

Comment:
nit: format as `// TODO(sherrypra): Implement this method.`

Including your name so later reader will know who thought they were on the hook for it, the ` ` after `//` is just formatting cleanup, the rest is to phrase the comment as a complete sentence, including punctuation, so it reads more easily to folks (this is part of Google style guides generally).

Reply:
Added!